src/core.js

```diff
@@ -1837,7 +1835,6 @@ define([
                             // unconditionally overwrite if no xform to compare
                             _this.send(formText, 'full');
                         } else {
-                            _this.closeModal();
```

Comment:
clean up. https://github.com/dimagi/Vellum/blob/master/src/core.js#L670

I would like to clean up all the random modals in src/core and put them into their own module. That would make it a lot easier to use these in other areas (like adding a waiting dialog to any future ajax calls)